### PR TITLE
Bug 1933184: Add maxUnavailable to DaemonSets

### PR DESCRIPTION
**Pull request by jsafrane**
To speed up cluster upgrade by allowing more nodes to be updated
simultaneously.

### CODE REVIEW

File: assets/node.yaml [hunk 1]
```diff
@@ -10,6 +10,10 @@ spec:
     matchLabels:
       app: openstack-manila-csi
       component: nodeplugin
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
```
Comment (wking) [inline assets/node.yaml:16]: `RollingUpdate` with `maxUnavailable: 1` is the default.  [The bug][1] is asking for `maxUnavailable: 10%`, so rollout time remains bounded as the node count grows.

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1933184#c0

Comment (jsafrane) [reply assets/node.yaml]: Fixed

**Pull request merged**